libraries/chain/webassembly/privileged.cpp

```diff
@@ -230,7 +230,7 @@ namespace eosio { namespace chain { namespace webassembly {
       cfg.validate();
       context.db.modify( context.control.get_global_properties(),
          [&]( auto& gprops ) {
-              gprops.configuration = cfg;
+              gprops.configuration.copy_from_v0(cfg);
```

Comment:
> Why do you need an assignment operator from v0 that leaves v1 and v2 data alone? 

This might be why. Does this change technically cause a consensus violation? The old code would leave `max_action_return_value_size` unchanged but this code will reset it to `config::default_max_action_return_value_size`. (I don't see any reason one can't call `set_blockchain_parameters_packed()` after activating `ACTION_RETURN_VALUE`)

Reply:
Good catch! Will revert.

Reply:
I don't think you need to entirely revert but rather `copy_from_v0()` shouldn't stomp on v1 & v2 items.

Reply:
> I don't think you need to entirely revert

(pending the style question in the other thread)